src/scenes/UserDetail/components/UserProfile.js

```diff
@@ -216,152 +293,203 @@ export default function UserProfile(props) {
                 spacing={3}
             >
                 {header}
-                {editToggle}
+                {editNameToggle}
             </Stack>
             <Divider />
+            {oldState.contact && (
+                <Stack
+                    direction={"row"}
+                    justifyContent={"space-between"}
+                    alignItems={"top"}
+                    spacing={3}
+                >
+                    <Typography>Contact Information</Typography>
+
+                    {editContactToggle}
+                </Stack>
+            )}
             <Box sx={{ width: "100%" }}>
-                {Object.keys(fields).map((key) => {
-                    if (editMode) {
-                        return (
-                            <TextField
-                                value={state[key]}
-                                variant={"standard"}
-                                fullWidth
-                                label={fields[key]}
-                                id={key}
-                                onChange={(e) => {
-                                    toChange.current = {
-                                        ...toChange.current,
-                                        [key]: e.target.value,
-                                    };
-                                    setState({
-                                        ...state,
-                                        [key]: e.target.value,
-                                    });
-                                }}
-                            />
-                        );
-                    } else {
+                <Stack direction={"row"} justifyContent={"space-between"}>
+                    <Typography>{fields.name}</Typography>
+                    <Typography>{oldState.name}</Typography>
```

Comment:
@duckbytes On the user I was logged in as, it didn't have any contact information, thus the edit button won't show but their name is still rendered. Is this the expected logic?
<img width="1269" alt="Screen Shot 2022-05-17 at 4 12 32 AM" src="https://user-images.githubusercontent.com/51278228/168890592-1e943d77-161b-4351-8050-6f5f4b246891.png">

Reply:
@LamVu1 Just my 2c here. I think the edit button should be present even if there isn't any contact information, otherwise the user can't add it. I'm not aware of any other way of adding this information in the application do correct me if I'm wrong.

Reply:
Hey @LamVu1 

This happens if the `contact` field is specifically null instead of just empty fields.

When adding a new user to the system normally, the contact field should never be null because the add new user lambda saves the user's email address.

The reason the offline user doesn't have this information is simply because I didn't add an extra line in the function that generates fake data to give that user contact info!

There could be some logic added to deal with the situation of that field being null, but I don't think it's really necessary. If that ever happens it means something else went wrong during the user's registration.